### ISSUE TITLE
cannot switch between text and tile version and keep the changes in text

### ISSUE THREAD

**Issue by JensLincke** [timfel/etoys-spur#11]

**Comment by timfel** [comment]
But do you get the warning? there should be a warning when you modified the text and switch back.

**Comment by JensLincke** [comment]
Yes, the warning comes... This is more like a feature request. There was
also sometimes some thing called UniTiles which could represent all
smalltalk...
